docs/source/api/notebooks/Notebook_6_API2CLI.ipynb

```diff
@@ -0,0 +1,336 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# in demo_run_train.py file\n",
+    "\n",
+    "from graphstorm.config import GSConfig\n",
+    "\n",
+    "......\n",
+    "\n",
+    "def fit(gs_args, cust_args):\n",
+    "    # Utilize GraphStorm's GSConfig class to accept arguments\n",
+    "    config = GSConfig(gs_args)\n",
+    "\n",
+    "    gs.initialize(ip_config=config.ip_config, backend=config.backend, local_rank=config.local_rank)\n",
+    "    acm_data = gs.dataloading.GSgnnData(part_config=config.part_config)\n",
+    "\n",
+    "    ......\n",
+    "\n",
+    "    model = RgatNCModel(g=acm_data.g,\n",
+    "                        num_heads=config.num_heads,\n",
+    "                        num_hid_layers=config.num_layers,\n",
+    "                        node_feat_field=config.node_feat_name,\n",
+    "                        hid_size=config.hidden_size,\n",
+    "                        num_classes=config.num_classes,\n",
+    "                        encoder_type=cust_args.rgat_encoder_type)   # here use the customized argument instead of GSConfig\n",
+    "\n",
+    "    ......"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# in demo_run_infer.py file\n",
+    "\n",
+    "from graphstorm.config import GSConfig\n",
+    "\n",
+    "......\n",
+    "\n",
+    "def infer(gs_args, cust_args):\n",
+    "    # Utilize GraphStorm's GSConfig class to accept arguments\n",
+    "    config = GSConfig(gs_args)\n",
+    "\n",
+    "    ......\n",
+    "\n",
+    "    model = RgatNCModel(g=acm_data.g,\n",
+    "                        num_heads=config.num_heads,\n",
+    "                        num_hid_layers=config.num_layers,\n",
+    "                        node_feat_field=config.node_feat_name,\n",
+    "                        hid_size=config.hidden_size,\n",
+    "                        num_classes=config.num_classes,\n",
+    "                        encoder_type=cust_args.rgat_encoder_type)   # here use the customized argument instead of GSConfig\n",
+    "\n",
+    "    model.restore_model(config.restore_model_path)\n",
+    "\n",
+    "    ......"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run CLIs on a Distributed Cluster  \n",
+    "\n",
+    "It is easy to modify the CLIs in the above cell to run them on a [Distributed clusters](https://graphstorm.readthedocs.io/en/latest/cli/model-training-inference/distributed/cluster.html). We need conduct three additional operations:\n",
+    "\n",
+    "1. Partition the ACM data in multiple partitions, e.g., 2 partition, and record its JSON file path, e.g., `./acm_gs_2p/acm.json`.\n",
```

Comment:
How to partition it? Any reference?

Reply:
Updated with link to `User Your Own Data` graph construction command.